src/api.js

```diff
@@ -1,12 +1,22 @@
 import axios from "axios";
-import {BACKEND_BASE_URI, COMPANIES_API_PATH} from "./constants";
+import { BACKEND_BASE_URI, COMPANIES_API_PATH } from "./constants";
 
 /*
   Fetch list of all companies from backend
  */
 export const fetchCompanies = async () => {
   try {
-    const response = await axios.get(`${BACKEND_BASE_URI}${COMPANIES_API_PATH}`);
+    const formatter = new Intl.NumberFormat("en-US", {
```

Comment:
This separates thousands with commas, whereas the task described in the issue is to separate thousands with spaces.

This is important, because in real life projects, the task description is given to you by product manager. You must thoroughly understand the task and give a solution that solves it.

In this situation you solved the different problem - not the one that was specified in the issue description.

Reply:
Thank you I will make the appropriate adjustments.